### PR TITLE
Update to support Laravel 5.4

### PR DESCRIPTION
**Pull request by daryledesilva**
- changed all `getToken()` to `token()`

### CODE REVIEW

**Comment by tortuetorche** [comment]
Hi @daryledesilva,

Your pull-request break compatibilities with Laravel < 5.4
See https://github.com/patricktalmadge/bootstrapper/pull/318 for a patch which is compatible with all versions of Laravel 5.*.

Cheers,
Tortue Torche

**Comment by daryledesilva** [comment]
@tortuetorche Added. Thanks for the feedback!

File: src/Bootstrapper/BootstrapperL5ServiceProvider.php [hunk 1]
```diff
@@ -207,7 +207,7 @@ function ($app) {
                     $app->make('collective::html'),
                     $app->make('url'),
                     $app->make('view'),
-                    $app['session.store']->getToken()
+                    (method_exists($app['session.store'], 'getToken') ? $app['session.store']->getToken() : $app['session.store']->token())
```
Comment (tortuetorche) [inline src/Bootstrapper/BootstrapperL5ServiceProvider.php]: To pass PSR-2 validation, you should do this:
```php
        $this->app->bind(
            'bootstrapper::form',
            function ($app) {
                $tokenMethod = method_exists($app['session.store'], 'getToken') ? 'getToken' : 'token';
                $form = new Form(
                    $app->make('collective::html'),
                    $app->make('url'),
                    $app->make('view'),
                    $app['session.store']->$tokenMethod()
                );

                return $form->setSessionStore($app['session.store']);
            },
            true
        );
```

**Comment by tortuetorche** [comment]
@daryledesilva Did you see my comment https://github.com/patricktalmadge/bootstrapper/pull/321#pullrequestreview-27898169 ?

**Comment by daryledesilva** [comment]
@tortuetorche okay doing now. will push commit in a few minutes

**Comment by daryledesilva** [comment]
@tortuetorche pushed changes. please check if this passes psr2 validation. thanks!

**Comment by daryledesilva** [comment]
thanks @PatrickRose !

**Pull request merged**